jwst/ramp_fitting/ramp_fit.py

```diff
@@ -229,16 +231,21 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
 
     # Get readnoise array for calculation of variance of noiseless ramps, and
     #   gain array in case optimal weighting is to be done
-    readnoise_2d, gain_2d = utils.get_ref_subs(model, readnoise_model, 
+    readnoise_2d, gain_2d = utils.get_ref_subs(model, readnoise_model,
                             gain_model)
 
     # Flag any bad pixels in the gain
     pixeldq = utils.reset_bad_gain( pixeldq, gain_2d )
 
-    # loop over data integrations
+    # In this 'First Pass' over the data, loop over integrations and data
+    #   sections to calculate the estimated median slopes, which will be used
```

Comment:
I think it's worth mentioning here that the actual slopes are also computed.

Reply:
Agreed, updated.